diff --git a/node_modules/protobufjs-cli/targets/static.js b/node_modules/protobufjs-cli/targets/static.js
index 3b4cf05..186ac27 100644
--- a/node_modules/protobufjs-cli/targets/static.js
+++ b/node_modules/protobufjs-cli/targets/static.js
@@ -817,7 +817,7 @@ function buildEnum(ref, enm) {
     push("");
     var comment = [
         enm.comment || enm.name + " enum.",
-        enm.parent instanceof protobuf.Root ? "@exports " + escapeName(enm.name) : "@name " + exportName(enm),
+        "@name " + exportName(enm),
         config.forceEnumString ? "@enum {string}" : "@enum {number}",
     ];
     Object.keys(enm.values).forEach(function(key) {
